src/components/ChatLog.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import ChatEntry from './ChatEntry';
+import PropTypes from 'prop-types';
+import './ChatLog.css';
+
+const ChatLog = ({ entries, onUpdateChatData, localName }) => {
+  const getChats = (entries) => {
```

Comment:
Totally fine to use this pattern of creating a function that returns an array of React elements, since it is shown in Learn; but I have not found it to be commonly used in practice.